Test/DependencyCollector/FunctionalTests/FuncTest/HttpTests.cs

```diff
@@ -389,7 +389,7 @@ public void TestRddForWin32ApplicationPool()
                             DeploymentAndValidationTools.SleepTimeForSdkToSendEvents);
 
                     var httpItems =
-                        allItems.Where(i => i.data.baseData.dependencyKind == DependencyKind.Http).ToArray();
+                        allItems.Where(i => i.data.baseData.type == "Http").ToArray();
```

Comment:
Repeating strings like "Http" should be in consts so there is only one copy to maintain.